__tests__/components/AnonymousChat/child/ChatItemMyTextV2.test.tsx

```diff
@@ -1,34 +1,173 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable global-require */
+/* eslint-disable react/display-name */
 import * as React from 'react';
-import {render} from '@testing-library/react-native';
+import FastImage from 'react-native-fast-image';
+import {act, fireEvent, render} from '@testing-library/react-native';
 
 import ChatItemMyTextV2 from '../../../../src/components/AnonymousChat/child/ChatItemMyTextV2';
+import {Context} from '../../../../src/context';
+import {DEFAULT_PROFILE_PIC_PATH} from '../../../../src/utils/constants';
 
-describe('TESTING ChatItemTargetText', () => {
-  it('RENDER not continuous should match snapshot', () => {
-    const {toJSON} = render(
-      <ChatItemMyTextV2
-        avatar="https://www.google.com"
-        username="username"
-        isContinuous={false}
-        message="message"
-        time="time"
-      />
+jest.spyOn(React, 'useRef').mockReturnValue({current: {close: jest.fn()}});
+jest.mock('react-native-reanimated', () => ({
+  ...jest.requireActual('react-native-reanimated'),
+  useSharedValue: jest.fn(() => ({value: 0})),
+  useAnimatedStyle: jest.fn(() => ({})),
+  View: require('react-native/Libraries/Components/View/View').View,
+  Animated: {
+    View: require('react-native/Libraries/Components/View/View').View
+  }
+}));
+jest.mock('react-native-gesture-handler', () => {
+  const originalModule = jest.requireActual('react-native-gesture-handler');
+  const {View} = require('react-native');
+  const React = require('react');
+
+  return {
+    ...originalModule,
+    Swipeable: React.forwardRef(({children, ...props}, ref) => {
+      React.useImperativeHandle(ref, () => ({
+        close: jest.fn()
+      }));
+
+      return <View {...props}>{children}</View>;
+    })
+  };
+});
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Mocking the `Swipeable` component from `react-native-gesture-handler` is a good practice for testing. However, it's important to ensure that the mocked component behaves as closely as possible to the real one. In this case, the mock doesn't seem to handle any swipe gestures, which might lead to inaccurate test results if your tests rely on such interactions.



<!-- This is an auto-generated comment by OSS CodeRabbit -->